sources/core/objects/Grille.cpp

```diff
@@ -37,15 +37,36 @@ void Grille::AttackBy(IdPtr<Item> item)
             SetPassable(D_ALL, Passable::FULL);
             cutted_ = true;
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
+            return;
         }
         else
         {
             Create<IOnMapObject>(Rod::T_ITEM_S(), GetOwner());
             Delete();
+            return;
         }
     }
+    Breakable::AttackBy(item);
```

Comment:
I have to change attackby(0) in human maybe I can make it attackbyhand() function.